packages/istanbul-lib-source-maps/test/transformer.test.js

```diff
@@ -80,4 +97,29 @@ describe('transformer', function() {
             }
         });
     });
+
+    it('maps each file only once, /path/to/file.js and \\path\\to\\file.js are the same file', function() {
+        if (isWindows()) {
+            return this.skip();
```

Comment:
I'm just asking because I don't know: should this be skipped on windows? Don't windows paths have `\\`s in them?

Reply:
A classic case of cargo cult programming, I don't know if it's needed or not,  I included it because it's also used in the 'maps statement locations' test 😉The test data includes windows style data, it doesn't rely on executing on windows, I'll try running the test without it!

Reply:
Yep, I tried, this check is necessary!

Reply:
@coreyfarrell @bcoe thoughts?

Reply:
These lines seem to the relevant ones:

```
  transformer
    1) maps statement locations
    - maps each file only once, /path/to/file.js and \path\to\file.js are the same file
  4 passing (15ms)
  1 pending
  1 failing
  1) transformer
       maps statement locations:
     TypeError: Cannot read property 'statementMap' of undefined
      at Context.<anonymous> (test\transformer.test.js:85:57)
npm ERR! Test failed.  See above for more details.
lerna ERR! npm test exited ENOENT in 'istanbul-lib-source-maps'
```

Reply:
The build is for Node.js v8.15.0 and it also complains about "ES6 feature [undefined] is not available in this environment" right after listing coverage stats.

Reply:
I can reproduce this error in Parallels on my mac, I'll try to figure it out.

Reply:
It's a windows path issue, the input path '/path/to/file.js' comes back as 'c:\path\to\files.js' on windows which makes `mapped.data[testDataSlash.coverageData.path]` undefined...

I've tried resolving this by creating test data paths manually (using `__dirname`, `path.join` and a regex replace), which makes the paths (and slashes) look and behave consistently in the tests on all platforms ☺️

Using `path.resolve` and `path.normalize` didn't work because the paths get converted to what the current os expects, not what the tests expect.

Reply:
Have you tried `path.posix.join` and such?

https://nodejs.org/dist/latest-v6.x/docs/api/path.html#path_path_posix

Reply:
@coreyfarrell, it looks much better now, short and elegant!